packages/webview_flutter/example/test_driver/webview_flutter_e2e.dart

```diff
@@ -576,6 +576,31 @@ void main() {
       expect(currentUrl, 'https://www.google.com/');
     });
 
+    testWidgets('onReceivedError', (WidgetTester tester) async {
+      final Completer<Map<String, String>> errorCompleter =
+          Completer<Map<String, String>>();
+
+      await tester.pumpWidget(
+        Directionality(
+          textDirection: TextDirection.ltr,
+          child: WebView(
+            key: GlobalKey(),
+            initialUrl: 'https://www.notawebsite..com',
+            onReceivedError: (String errorCode, String description) {
+              errorCompleter.complete(<String, String>{
+                'errorCode': errorCode,
+                'description': description,
+              });
+            },
+          ),
+        ),
+      );
+
+      final Map<String, String> errorResult = await errorCompleter.future;
+      expect(errorResult['errorCode'], isNotNull);
+      expect(errorResult['description'], isNotNull);
+    });
```

Comment:
Let's add a test with a valid url that verifies the error callback is not invoked on a valid url. I'd use a data url like: 

https://github.com/flutter/plugins/blob/c03166bc12791bebc172b63259bd7200c052187e/packages/webview_flutter/example/test_driver/webview_flutter_e2e.dart#L116

Reply:
done